pkg/security/tests/chown_test.go

```diff
@@ -254,3 +256,230 @@ func TestChown(t *testing.T) {
 	})
 
 }
+
+func TestChownUserGroup(t *testing.T) {
+	SkipIfNotAvailable(t)
+	if checkUserGroupAvailable() != nil {
+		t.Skip("User/group needed tools not available")
+	}
+
+	testUser := "test_user_1"
+	testUID := int32(1901)
+	testGroup := "test_group_1"
+	testGID := int32(1902)
+	ruleDefs := []*rules.RuleDefinition{
+		{
+			ID:         "test_rule",
+			Expression: `chown.file.path == "{{.Root}}/test-chown" && chown.file.destination.user == "` + testUser + `" && chown.file.destination.group == "` + testGroup + `"`,
+		},
+		{
+			ID:         "test_rule2",
+			Expression: `chown.file.path == "{{.Root}}/test-symlink" && chown.file.destination.user == "` + testUser + `" && chown.file.destination.group == "` + testGroup + `"`,
+		},
+		{
+			ID:         "test_rule3",
+			Expression: `chown.file.path == "{{.Root}}/test-chown" && chown.file.destination.user == "` + testUser + `" && chown.file.destination.gid == -1`,
+		},
+		{
+			ID:         "test_rule4",
+			Expression: `chown.file.path == "{{.Root}}/test-chown" && chown.file.destination.uid == -1 && chown.file.destination.group == "` + testGroup + `"`,
+		},
+	}
+
+	currentUser, err := user.Current()
+	if err != nil {
+		t.Fatal(err)
+	}
+	currentGroup, err := user.LookupGroupId(currentUser.Gid)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// create temporary user/group
+	removeUser(testUser)   // just in case
```

Comment:
I would rather use other passwd/group files. We could use an environment variable to pass these files to the user/group resolver.

Reply:
Yep I agree. Suggestion applied on last commit, thank you!